website/content/docs/datasources/http/http.md

```diff
@@ -0,0 +1,49 @@
+---
+description: |
+  The http Data Source retrieves information from an http endpoint to be used
+  during Packer builds
+page_title: Http - Data Sources
+---
+
+<BadgesHeader>
+  <PluginBadge type="official" />
+  <PluginBadge type="hcp_packer_ready" />
+</BadgesHeader>
+
+# Http Data Source
+
+Type: `http`
+
+The `http` Data Source retrieves information from an http endpoint to be used
+during packer builds
```

Comment:
Copying and pasting the terraform datasource docs because I think the wording was a little better. What do you think?
```suggestion
The `http` data source makes an HTTP GET request to the given URL and exports information about the response.
```

Reply:
Yeah that sounds better